runtimes/node-18.0/server.js

```diff
@@ -62,30 +88,28 @@ const server = micro(async (req, res) => {
                 return this.send('', statusCode, headers);
             }
         },
-        log: function () {
-            const args = [];
-            for (const arg of Array.from(arguments)) {
-                if (arg instanceof Object || Array.isArray(arg)) {
-                    args.push(JSON.stringify(arg));
-                } else {
-                    args.push(arg);
-                }
+        log: function (message) {
+            if (message instanceof Object || Array.isArray(message)) {
+                logs.push(JSON.stringify(message));
+            } else {
+                logs.push(message + "");
             }
-            logs.push(args.join(" "));
         },
-        error: function () {
-            const args = [];
-            for (const arg of Array.from(arguments)) {
-                if (arg instanceof Object || Array.isArray(arg)) {
-                    args.push(JSON.stringify(arg));
-                } else {
-                    args.push(arg);
-                }
+        error: function (message) {
+            if (message instanceof Object || Array.isArray(message)) {
+                errors.push(JSON.stringify(message));
+            } else {
+                errors.push(message + "");
             }
-            errors.push(args.join(" "));
         },
     };
 
+    console.stdlog = console.log.bind(console);
+    console.stderror = console.error.bind(console);
+    console.stdinfo = console.info.bind(console);
+    console.stddebug = console.debug.bind(console);
+    console.stdwarn = console.warn.bind(console);
```

Comment:
why are we still binding this ? haven't we stopped using it ?

Reply:
Developers will always use console.log. We still bind so we can see if they used console.log. If they did, we tell them (in log) that "This is not supported, please use context.log() instad".

Reply:
okay 👍